packages/package-builder/src/Reflection/PrivatesAccessor.php

```diff
@@ -24,6 +42,21 @@ public function getPrivateProperty(object $object, string $propertyName)
         return $propertyReflection->getValue($object);
     }
 
+    /**
+     * @template T
+     *
+     * @param class-string<T> $valueClassName
+     * @param T $value
+     */
+    public function setPrivatePropertyOfClass(object $object, string $propertyName, mixed $value, string $valueClassName): void
+    {
+        if (!($value instanceof $valueClassName)) {
```

Comment:
parentheses seems can be removed:

```suggestion
        if (! $value instanceof $valueClassName) {
```